docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2417 @@
+{
```

Comment:
<p>great description. Capitalize <code>Units</code> and <code>Electrodes</code>  table names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>